examples/function-comment-bot/src/tools/show-publish.ts

```
@@ -0,0 +1,29 @@
+import { client, spaceId, environmentId, entryId } from './contentful-client-and-imports';
+
+export const createComment = async () => {
+  if (!entryId) {
+    console.error('No entry ID provided');
+    return;
+  } else {
+    try {
+      const comment = await client.comment.create(
+        {
+          spaceId,
+          environmentId,
+          entryId,
+        },
+        {
+          body: '/show-publish',
+          status: 'active',
+        }
+      );
+
+      console.log('Comment created');
+      console.dir(comment, { depth: 5 });
+    } catch (error) {
+      console.error(error);
```

should this throw, or do we want to just swallow the error?

---

I think it's ok to swallow the error here, since it's just a script that you'll run using `npm run show-publish` and you see the output of the error in the terminal when you are running the command. But happy to hear other suggestions/thoughts on this!